neurom/fst/_bifurcationfunc.py

```diff
@@ -51,45 +53,52 @@ def skip_0_length(sec):
 
         return cur
 
-    ch0, ch1 = (skip_0_length(bif_point.children[0].points),
-                skip_0_length(bif_point.children[1].points))
+    if len(bif_point.children) == 2:
```

Comment:
This means that if it's not a bifurcation, it's silently ignored, which I would find surprising, especially since there isn't (currently) a guaranteed ordering: If the children list gets shuffled, you'd get different results since it's only the first two children that are looked at.

I think doing an `assert len(bif_point.children) == 2, 'A bifurcation must have only 2 children'` which would raise an exception when it finds a multifurcation.

Alternative, having a warning printed would work, if non-bifurcations come up often, but aren't a show-stopper.